jetty-hazelcast/src/main/java/org/eclipse/jetty/hazelcast/session/HazelcastSessionDataStoreFactory.java

```diff
@@ -63,6 +63,8 @@
 
     private Config serverConfig;
 
+    private boolean autoDetectionConfigDisabled;
+
```

Comment:
```suggestion
private boolean autoDetectionConfigEnabled = false;
```

Reply:
I choose the name to leave the option enable per default.
I would definitely prefer leave it `enabled` per default as it is the default strategy to discovering other hazelcast servers running in the same network  and so creating automatically some clusters. 
But is causing an issue in the kubernetes test environment.

Reply:
the issue is here for long time :) (since 4.1)
just happening more often recently.
https://hazelcast.com/blog/hazelcast-discovery-auto-detection/

Reply:
basically this one of the key feature here. when the instance starts, it auto connect to other hazelcast instance in the same network to share data. having it per default is better but nor for testing

Reply:
So it can still be called `autoDetectionConfigEnabled` but be `true` by default, and have the CI config set it to `false`?

Reply:
I usually prefer having boolean with default `false` as jvm default...
as this parameter was really for test purpose for one single test and could potentially create some confusions for our users  I removed and use a single configuration file for the test.